tests/unit/test_charm.py

```diff
@@ -1,13 +1,13 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-
 import unittest
+from typing import List
```

Comment:
Which Python version is being targeted by the charm? If it is 3.10 or later `list` can be used instead of `typing.List` which is deprecated